### ISSUE TITLE
PropTypes are deprecated (and make the library crash with react-native-web)

### ISSUE THREAD

**Issue by alessiocancian** [xotahal/react-native-material-ui#500]
This library uses deprecated __propTypes__ from `react-native` which aren't exported by `react-native-web` (https://github.com/necolas/react-native-web/issues/1543).

As a result when trying to use `react-native-material-ui` on web I got this error:
```
Uncaught TypeError: Cannot read property 'style' of undefined
   at eval(webpack-internal:///./node_modules/react-native-material-ui/src/IconToggle/IconToggle.react.js)
   at Object../node_modules/react-native-material-ui/src/IconToggle/IconToggle.react.js
   ....
```

I'm sending a PR with the patch I used to fix it...

**Comment by samaneh-kamalian** [comment]
> This library uses deprecated **propTypes** from `react-native` which aren't exported by `react-native-web` ([necolas/react-native-web#1543](https://github.com/necolas/react-native-web/issues/1543)).
> 
> As a result when trying to use `react-native-material-ui` on web I got this error:
> 
> ```
> Uncaught TypeError: Cannot read property 'style' of undefined
>    at eval(webpack-internal:///./node_modules/react-native-material-ui/src/IconToggle/IconToggle.react.js)
>    at Object../node_modules/react-native-material-ui/src/IconToggle/IconToggle.react.js
>    ....
> ```
> 
> I'm sending a PR with the patch I used to fix it...

same problem!

**Comment by samaneh-kamalian** [comment]
Excuse me, I have a serious problem with this. Is there no solution?

**Comment by alessiocancian** [comment]
> Excuse me, I have a serious problem with this. Is there no solution?

Yes, you can patch the package using patch-package, see my PR with the fix
